src/Features/Core/Portable/GenerateMember/GenerateParameterizedMember/AbstractGenerateParameterizedMemberService.CodeAction.cs

```diff
@@ -52,8 +52,7 @@ private partial class GenerateParameterizedMemberCodeAction : CodeAction
                             isAbstract ? FeaturesResources.Generate_abstract_method_1_0 : FeaturesResources.Generate_method_1_0;
 
                         var name = state.IdentifierToken.ValueText;
-                        var destination = state.TypeToGenerateIn.Name;
-                        return string.Format(text, name, destination);
+                        return string.Format(text, name);
```

Comment:
this seems broken.  the resource strings have 1_0 in them, indicating they take two args.  But you're only passing in one.